### DISCUSSION TITLE
When to start this project? Again?

### DISCUSSION THREAD

**Discussion by rNLKJA** [crackerjack-uniRM/2021-MAST30034-ADS-Battery-Optimisation#5] [Q&A] [answered]
As the question.

**Comment by rNLKJA** [comment, marked as answer]
Well, I think this will postpone to July. Let's hope we could start the project on that day.

Also, we might re-start the New York Taxi project again for @QUZIHANWU ADS assignment preparation.

**Reply by QUZIHANWU** [reply]
yep
